object-cache.php

```diff
@@ -937,6 +968,19 @@ protected function _should_persist( $group ) {
 	}
 
 	/**
+	 * Should this group use Redis hashes?
+	 *
+	 * @param string $group Cache group.
+	 * @return bool True if the group should use Redis hashes, false if not.
+	 */
+	protected function _should_use_redis_hashes( $group ) {
```

Comment:
@danielbachhuber you recently had a PR that removed the preceding `_` for a protected function. Let's stick with that convention.